src/app.js

```diff
@@ -2,0 +1,65 @@
+/* eslint-disable no-console */
+import fs from 'fs/promises';
+
+const sourceFilePath = process.argv[2];
+const destinationFilePath = process.argv[3];
+
+const isDirectory = async (path) => {
+  try {
+    const stats = await fs.stat(path);
+
+    return stats.isDirectory();
+  } catch (error) {
+    console.error(error);
+  }
+};
+
+const copyFile = async (sourcePath, destinationPath) => {
+  // console.log('start');
+  //
+  if (!destinationPath) {
+    console.error('no second param');
+
+    return;
+  }
+
+  try {
+    if (await isDirectory(sourcePath)) {
+      console.error('sourcePath is the directory');
+    }
+  } catch (error) {
+    console.log(error);
+  }
+
+  try {
+    if (await isDirectory(destinationPath)) {
+      console.error('destinationPath is the directory');
```

Comment:
Consider providing a more descriptive error message here to help with debugging.